instructure/canvas-ios
# Fix grades for previously graded submissions

refs: MBL-13454
affects: student
release note: Fixed a bug that would cause an assignment to incorrectly appear as submitted

Test plan:
* See steps [in ticket](https://instructure.atlassian.net/browse/MBL-13454)
* From the assignments in the ticket, only the quiz should have a grade on assignment details
* All other assignments in the ticket should not show the Grade section
* View submitted, excused, graded, late, and unsubmitted assignments using another student (narmstrong > s > CS 1400 has examples)
